src_c/display.c

```diff
@@ -1248,8 +1257,20 @@ pg_flip(PyObject *self, PyObject *args)
     }
     Py_END_ALLOW_THREADS;
 
-    if (status == -1)
-        return RAISE(pgExc_SDLError, SDL_GetError());
+    if (status == -1) {
```

Comment:
This isn't directly related to the changes you made, but I noticed this in the surrounding code while reviewing your update.

The `status` variable is initialized to 0 and is only ever set on line 1255:
`status = SDL_UpdateWindowSurface(win) == -1;`

Based on this, `status` can only be 0 or 1 and the check on line 1260 never evaluates to true. Also, any check of `SDL_UpdateWindowSurface()`'s return code should not just be checking `== -1`, it should be `<= -1` as the [SDL_UpdateWindowSurface's documentation](https://wiki.libsdl.org/SDL_UpdateWindowSurface) states:
> Returns 0 on success or a negative error code on failure

Reply:
Good catch.